Internship.API/Controllers/InternController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Internship.API.Models;
+using Internship.API.Services;
+using Internship.API.Services.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Internship.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class InternController : ControllerBase
```

Comment:
Can you change the controller name to "InternsController" to follow the standard?